PROVESFlightControllerReference/test/int/radio_test.py

```
@@ -34,6 +34,15 @@ def setup_test(fprime_test_api: IntegrationTestAPI, start_gds):
     )
 
 
+def test_00_setup_only(fprime_test_api: IntegrationTestAPI, start_gds):
```

Ruff (0.15.12)

[warning] Unused function argument: `start_gds`

(ARG001)